### PR TITLE
n19825 announcement, notes and questions

### PR DESCRIPTION
**Pull request by jonatack**

### CODE REVIEW

**Comment by jonatack** [comment]
Pinging @dhruv for review.

File: _posts/2021-01-13-#19825.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+      their addresses are not gossiped.
+- Neither banning nor discouragement protects against DoS attacks as IP
+  addresses are inexpensive.
+- Programatic banning or disconnection increases the risk of a network split.
+- The `BanMan` interface allows banning and unbanning via `CNetAddr` and
+  `CSubNet`. This PR consolidates to only accept `CSubNet` (an IP address is a
+  subnet of 1).
+- This PR also adds two new RPC commands:
+	1. `setban listbanned ip`: lists all the ban entries that include `ip`
+	2. `setban removeall ip`: removes all ban entries that include `ip`
```
Comment (jonatack) [inline _posts/2021-01-13-#19825.md:33]: will change the tabs to spaces in these two lines

**Review by jonatack — COMMENTED** [review]
@dhruv I made a number of formatting and small content fixups from the source code on slack.

Fixing the comment below, adding the commit, and will push to get this up. If there are any further edits to be done, an new PR can be opened.

File: _posts/2021-01-13-#19825.md (hunk 1 above)
Comment (jonatack) [reply _posts/2021-01-13-#19825.md]: made some other fixups to my first push

**Comment by jonatack** [comment]
For the next edits here, "programatic" is missing a "t" in two places.

File: _posts/2021-01-13-#19825.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+    2. `setban removeall ip`: removes all ban entries that include `ip`
+
+## Questions
+
+1.  What are some use cases for banning specific IP addresses or subnets?
+
+2.  Is programatic banning/disconnection an effective DoS countermeasure for
+    misbehaving peers?
+
+3.  How can programatic banning/disconnection be abused by an attacker?
```
Comment (jonatack) [inline _posts/2021-01-13-#19825.md:46]: s/programatic/programmatic|automatic/ here and in 2 other places

Comment (jnewbery) [reply _posts/2021-01-13-#19825.md:46]: I agree that automatic is better.

**Pull request merged**